cypress/support/TestListManager.js

```diff
@@ -3,225 +3,140 @@
-    describe(test.group, () =>
-    {
-      var currentGroupName = test.group
+    helpers.ConLog(funcName, `Adding Group: ${test.group}`);
+    describe(test.group, () => {
+      var currentGroupName = test.group;
       while (test != undefined && test.group == currentGroupName)
       {
-        helpers.ConLog(funcName, `Adding Test Case: ${test.name}`)
-        it(test.name, test.func)
-        test = testListIterator.next
+        helpers.ConLog(funcName, `Adding Test Case: ${test.name}`);
+        it(test.name, test.func);
+        test = testListIterator.next;
       }
     })
   }
 }
 
-class TestListIterator
-{
-  constructor(testList)
-  {
-    this.testList = testList
-    this.index = 0
-    this.currentGroup = {name: ''}
+class TestListIterator {
+  constructor(testList) {
+    this.testList = testList;
+    this.index = 0;
+    this.currentGroup = {name: ''};
   }
 
   // groupName.testName - 'testName' from 'groupName'
   // TODO: Add support for these wild card versions
   // *.*                - All Groups, All Tests
   // *.testName         - All tests with all groups matching 'testName'
   // groupName.*        - All tests from 'groupName'
-  get next()
-  {
-    if (this.index >= this.testList.length) return undefined
+  get next() {
+    if (this.index >= this.testList.length) return undefined;
 
-    var x = this.testList[this.index].split('.')
-    if (x.length != 2) throw `Invalid item in testList[${this.index}]: "${this.testList[this.index]}" - 'group DOT testName' format is expected`
-    var groupName = x[0]
-    var testName = x[1]
+    var x = this.testList[this.index].split('.');
+    if (x.length != 2) throw `Invalid item in testList[${this.index}]: "${this.testList[this.index]}" - 'group DOT testName' format is expected`;
+    var groupName = x[0];
+    var testName = x[1];
 
-    if (this.currentGroup.name != groupName)
-    {
-      this.currentGroup = GetTestGroup(groupName)
-      if (this.currentGroup == undefined) throw `Group '${groupName}' NOT found in testGroups`
+    if (this.currentGroup.name != groupName) {
+      this.currentGroup = GetTestGroup(groupName);
+      if (this.currentGroup == undefined) throw `Group '${groupName}' NOT found in testGroups`;
     }
     
-    var test = GetTest(this.currentGroup, testName)
-    if (test == undefined) throw `Test '${testName}' NOT found in test group '${groupName}'`
+    var test = GetTest(this.currentGroup, testName);
+    if (test == undefined) throw `Test '${testName}' NOT found in test group '${groupName}'`;
 
-    this.index++
-    return {group: this.currentGroup.name, name: test.name, func: test.func}
+    this.index++;
+    return {group: this.currentGroup.name, name: test.name, func: test.func};
   }
 }
 
-function GetTestGroup(name)
-{
-  for (var i = 0; i < testGroups.length; i++)
-  {
-    if (testGroups[i].name == name) 
-    {
-      return testGroups[i]
-    }
+function GetTestGroup(name) {
+  for (var i = 0; i < testGroups.length; i++) {
```

Comment:
I would avoid the `for` loops and instead use the safer array methods that clearly explain what your trying to do and reader can know what should happen in the loop. 

Example:
`testGroups.find(testGroup => testGroup.name === name)`

Reply:
fixed